<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainPage</class>
 <widget class="QWidget" name="MainPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>529</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="accountListLabel">
   <property name="geometry">
    <rect>
     <x>43</x>
     <y>23</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255,255,255);</string>
   </property>
   <property name="text">
    <string>账户</string>
   </property>
  </widget>
  <widget class="QLabel" name="totalBalanceLabel">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>22</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>451</x>
     <y>34</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(102, 102, 102);</string>
   </property>
   <property name="text">
    <string>总余额</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="addAccountBtn">
   <property name="geometry">
    <rect>
     <x>658</x>
     <y>427</y>
     <width>94</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton{color:white;border:1px solid  rgb(46,78,126);border-radius:6px;
background-color: rgb(46,78,126);}
QToolButton:hover{background-color:rgb(66,98,146);}
QToolButton:disabled{background-color:#cecece;}</string>
   </property>
   <property name="text">
    <string>创建账户</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="accountTableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>132</y>
     <width>759</width>
     <height>286</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget{color:rgb(102,102,102);border:none;background-color:rgb(22,24,28); }
QHeaderView{color:rgb(102,102,102);background-color:rgb(22,24,28);min-height:26;}
QHeaderView::section{background-color: rgb(22,24,28);border: none;
font: 12px &quot;微软雅黑&quot;;}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="autoScroll">
    <bool>true</bool>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>19</width>
     <height>12</height>
    </size>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::SolidLine</enum>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>账户</string>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>地址</string>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>余额/TV</string>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="loadingWidget" native="true">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>461</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(22,24,28);</string>
   </property>
   <widget class="QLabel" name="loadingLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>211</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(178, 178, 178);
background-color: rgb(22,24,28);</string>
    </property>
    <property name="text">
     <string>请稍等...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="initLabel">
   <property name="geometry">
    <rect>
     <x>206</x>
     <y>211</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(178, 178, 178);</string>
   </property>
   <property name="text">
    <string>您当前还没有账户，请添加</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="TvRaBtn">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>75</y>
     <width>81</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255,255,255);</string>
   </property>
   <property name="text">
    <string>TV</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="AcsRaBtn">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>75</y>
     <width>81</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255,255,255);</string>
   </property>
   <property name="text">
    <string>ACS</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="TdcRaBtn">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>75</y>
     <width>81</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255,255,255);</string>
   </property>
   <property name="text">
    <string>3DC</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="NbjRaBtn">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>75</y>
     <width>81</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255,255,255);</string>
   </property>
   <property name="text">
    <string>NBJ</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="NotRaBtn">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>75</y>
     <width>81</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255,255,255);</string>
   </property>
   <property name="text">
    <string>NOT</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
